core/src/main/scala/kafka/log/UnifiedLog.scala

```diff
@@ -1738,12 +1747,12 @@ object Log extends Logging {
             logDirFailureChannel: LogDirFailureChannel,
             lastShutdownClean: Boolean = true,
             topicId: Option[Uuid],
-            keepPartitionMetadataFile: Boolean): Log = {
+            keepPartitionMetadataFile: Boolean): UnifiedLog = {
     // create the log directory if it doesn't exist
     Files.createDirectories(dir.toPath)
-    val topicPartition = Log.parseTopicPartitionName(dir)
+    val topicPartition = UnifiedLog.parseTopicPartitionName(dir)
     val segments = new LogSegments(topicPartition)
-    val leaderEpochCache = Log.maybeCreateLeaderEpochCache(
+    val leaderEpochCache = UnifiedLog.maybeCreateLeaderEpochCache(
```

Comment:
Should we change the logging prefix to UnifiedLog in line 1760 too?

Reply:
Done. Thanks for catching this!